### ISSUE TITLE
Fix playlist if soundcloud / youtube is blocked by plugins

### ISSUE THREAD

**Issue by Illyism** [musicplayer-io/redditmusicplayer#24]
> Soundcloud still uses Flash, huh? I have Chrome set to click-to-play plugins, and it totally breaks the player - only works if I whitelist everything and come back to the site.

> If it's not too out of scope please figure something out about dead links (removed videos/tracks). Like a search link, or even an option to automatically play a replacement.

**Comment by akaleeroy** [comment]
It doesn't skip Soundcloud links requiring Flash in Chrome 43.0.2357.124 m set to Click-to-play plugins. I don't know whether something should be done about this tho, it's understandable. Maybe alert the user to whitelist the site?

Also doesn't skip dead Soundcloud links:
> `http://api.soundcloud.com/tracks/72416300.json?callback=jQuery214042667057318612933_1434986011112&client_id=5441b373256bae7895d803c7c23e59d9&_=1434986011118 Failed to load resource: the server responded with a status of 404 (Not Found)`

My suggestion would be to 
1) indicate dead link
then
2a) simply skip - important, users can't take precautions like whitelisting here
2b) offer a replacement by searching YouTube/Soundcloud for the keywords

Nice work mate, rock on!
